### ISSUE TITLE
[Bug] multi select also not work in library after single select and cancel

### ISSUE THREAD

**Issue by CJrZhang** [tachiyomiorg/tachiyomi#4189]
## Device information
* Tachiyomi version: 0.10.7 Stable
* Android version: 9
* Device: Across any devices

## Steps to reproduce
1. Enter your library
2. Long press select one manga and enter the active select mode
3. single select one manga and cancel it
4. Try to do multi select by long press another manga

### Expected behavior
Multi select will be done.

### Actual behavior
Unable to multi select, only work as doing single click select.

**Comment by arkon** [comment]
> single select one manga and cancel it
> 4. Try to do multi select by long press another manga

I'm not sure this makes sense. Can you maybe reword this?

**Comment by CJrZhang** [comment]
> > single select one manga and cancel it
> > 4. Try to do multi select by long press another manga
> 
> I'm not sure this makes sense. Can you maybe reword this?

Just like what is mentioned in issue #3769. In library we should be able to long press one manga and then another to multi select the mangas between them. But I am not able to do multi select after single tap selecting any manga in the library and then cancel this selection by tap it again. I don't know whether I state it clearly.
